server/api/utils/oauth_helper.py

```diff
@@ -0,0 +1,85 @@
+import base64
+from google_auth_oauthlib.flow import Flow
+from django.conf import settings
+from Crypto.Cipher import AES
+
+from Crypto import Random
+IV = Random.new().read(AES.block_size)
+KEY = bytes(settings.CRYPTOGRAPHIC_KEY, 'utf-8')
+CIPHER = AES.new(KEY, AES.MODE_CFB, IV)
+
+GOOGLE_CREDENTIALS = {
+    "web": {
+        "client_id": settings.GOOGLE_CLIENT_ID,
+        "project_id": "hini-1530850449245",
```

Comment:
Same applies here.

Reply:
I am sorry about this. Will fix immediately @BolajiOlajide